surch/organization.py

```diff
@@ -19,199 +19,200 @@
 
 import requests
 
-from . import logger, repo, utils, constants
-
-REPO_DETAILS_API_URL = \
-    'https://api.github.com/{0}/{1}/repos?type={2}&per_page={3}&page={4}'
-ORG_DETAILS_API_URL = 'https://api.github.com/{0}/{1}'
-
-
-lgr = logger.init()
+from . import repo, utils, constants
 
 
 class Organization(object):
     def __init__(
             self,
             search_list,
             organization,
-            git_user,
-            git_password,
+            verbose=False,
+            git_user=None,
+            results_dir=None,
+            git_password=None,
             print_result=False,
-            organization_flag=True,
             repos_to_skip=None,
             repos_to_check=None,
+            is_organization=True,
             consolidate_log=False,
-            cloned_repos_path=constants.CLONED_REPOS_PATH,
-            results_dir=constants.RESULTS_PATH,
-            verbose=False,
+            cloned_repos_dir=None,
             remove_cloned_dir=False,
             **kwargs):
-        """Surch instance define var from CLI or config file
+        """Surch org instance define var from CLI or config file
         """
+        utils.check_if_cmd_exists_else_exit('git')
+
+        self.logger = utils.logger
+        self.logger.setLevel(logging.DEBUG if verbose else logging.INFO)
         self.print_result = print_result
         self.search_list = search_list
         self.organization = organization
         self.results_dir = results_dir
         if repos_to_skip and repos_to_check:
```

Comment:
re-arrange all conditions to appear at the top of the function. this way reader can better comprehend what are the conditions that must be met in order to run surch.